mkosi.md

```diff
@@ -331,7 +331,8 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
 
 : The image format type to generate. One of `directory` (for generating OS
   images inside a local directory), `subvolume` (similar, but as a btrfs
-  subvolume), `tar` (similar, but a tarball of the image is generated), `cpio`
+  subvolume), `subvolume_ro` (btrfs read-only subvolume),
```

Comment:
What's the reasoning behind adding `subvolume_ro`? Couldn't this be exchanged for just doing `subvolume` and have people who need that set the read-only flag in a finalise script? I'm asking because the extra tracking of two different output formats seems like added complexity that might be good to avoid.